src/hermes/config.py

```diff
@@ -121,6 +129,10 @@ def init_logging():
     if _loggers:
         return
 
+    # Make sure the directories to hold the log files exists (or else create)
+    _config['logging']['handlers']['logfile']['filename'].parent.mkdir(exist_ok=True, parents=True)
+    _config['logging']['handlers']['auditfile']['filename'].parent.mkdir(exist_ok=True, parents=True)
+
```

Comment:
These will fail if there is something different than the default value configured (as this usually will be a string not a path...)

```suggestion
    # Make sure the directories to hold the log files exists (or else create)
    pathlib.Path(_config['logging']['handlers']['logfile']['filename']).parent.mkdir(exist_ok=True, parents=True)
    pathlib.Path(_config['logging']['handlers']['auditfile']['filename']).parent.mkdir(exist_ok=True, parents=True)

```